catkin_home/src/arm_vision/package.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>arm_vision</name>
+  <version>0.0.0</version>
+  <description>The arm_vision package</description>
```

Comment:
a longer description if possible, I see you are using a 'PCL' library add some links and explanation of whats going on.